crypto/asn1/a_object.c

```diff
@@ -85,7 +85,7 @@ int a2d_ASN1_OBJECT(unsigned char *out, int olen, const char *buf, int num)
             c = *(p++);
             if ((c == ' ') || (c == '.'))
                 break;
-            if (!isdigit(c)) {
+            if (!ossl_isdigit(c)) {
```

Comment:
This implies that all the code above should be ASCII, so all the character constants above should be ossl_to_ascii()'d